src/app/donation-start/donation-start-form/donation-tipping-slider/donation-tipping-slider.component.ts

```diff
@@ -0,0 +1,214 @@
+import { AfterContentInit, Component, ElementRef, Input, OnChanges, OnDestroy, OnInit, Renderer2, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'app-donation-tipping-slider',
+  templateUrl: './donation-tipping-slider.component.html',
+  styleUrls: ['./donation-tipping-slider.component.scss']
+})
+export class DonationTippingSliderComponent implements OnInit, AfterContentInit, OnChanges, OnDestroy {
+
+
+  @Input() spaceBelow: 0 | 1 | 2 | 3 | 4 | 5 | 6 = 0;
+
+  @Input() colourScheme:
+  | 'primary' // blue
+  | 'secondary' // green
+  | 'tertiary' // coral
+  | 'brand-1' // cc-red
+  | 'brand-2' // wgmf-purple
+  | 'brand-3' // gmf-green
+  | 'brand-4' // emf-yellow
+  | 'brand-5' // c4c-orange
+  | 'brand-6' // mhf-turquoise
+  | 'white'
+  | 'black'
+  | 'grey-extra-light'
+  | 'grey-light'
+  | 'grey-medium'
+  | 'grey-dark' = 'primary';
+
+  @Input() percentageCurrent: number;
+  @Input() percentageStart: number;
+  @Input() percentageEnd: number;
+  @Input() donationAmount: number;
+  //ISO-4217 currency code (e.g. GBP, USD)
+  @Input() donationCurrency!: 'GBP' | 'USD';
```

Comment:
Would love to get stronger types and make this more like:

```suggestion
  @Input() percentageCurrent: percentage;
  @Input() percentageStart: percentage;
  @Input() percentageEnd: percentage;
  @Input() donationAmount: donationAmount;
```

Percentages would be constrained to between zero and 100, donation amount would be constrained to between 1 and 25,000 and would encapsulate the currency and the number together. But definitely not for today!